docs/reference/indices/split-index.asciidoc

```diff
@@ -47,6 +49,27 @@ Splitting works as follows:
 * Finally, it recovers the target index as though it were a closed index which
   had just been re-opened.
 
+[float]
+=== Why doesn't Elasticsearch support incremental resharding?
+
+Going from `N` shards to `N+1` shards, aka. incremental resharding is indeed a
+feature that is supported by many key-value stores, which leverage the fact that
+consistent hashing only requires `1/N`-th of the keys to be relocated. However
+Elasticsearch's unit of storage, shards, are Lucene indices. Unfortunately,
+taking a significant portion of a Lucene index, be it only 5% of documents,
+deleting them and indexing them on another shard typically comes with a much
+higher cost than with a key-value store. This cost is kept reasonable when
+growing the number of shards by a multiplicative factor as described in the
+above section: this allows Elasticsearch to perform the split locally, which
+in-turn allows to perform the split at the index level rather than reindexing
+documents that need to move, as well as using hard links for efficient file
+copying.
+
+
+Moreover, only allowing large increments of the number of shards also has the
+good property of reducing the number of times that an index needs to be split
+during its lifetime.
```

Comment:
I don't like this paragraph all that much. It comes off as paternalistic. "You don't want this feature anyway." I figure we should just drop the paragraph and let users decide if they want the feature or not. If you feel like we have to keep the paragraph we should rework it so this sentiment comes across as a silver lining.

Reply:
You made a good point. I removed it.